Dockerfile.travis

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 Shift Devices AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM bitbox-wallet-dev
+ENV GOPATH /opt/go
+ENV GOROOT /opt/go_dist/go
+ENV PATH $GOROOT/bin:$GOPATH/bin:$PATH
+ENV PATH /opt/qt5/bin:$PATH
+
+RUN mkdir -p /opt/go/src/github.com/digitalbitbox/bitbox-wallet-app/
+ADD . /opt/go/src/github.com/digitalbitbox/bitbox-wallet-app/
+
+RUN make -C /opt/go/src/github.com/digitalbitbox/bitbox-wallet-app/ init
+RUN make -C /opt/go/src/github.com/digitalbitbox/bitbox-wallet-app/ qt-linux
```

Comment:
move down one line? better to see `ci` failures early, and the slower build that breaks less often after.